python/CMakeLists.txt

```diff
@@ -61,7 +61,8 @@ add_custom_command(OUTPUT ${PADDLE_PYTHON_BUILD_DIR}/.timestamp
     COMMAND ${CMAKE_COMMAND} -E touch ${PADDLE_PYTHON_BUILD_DIR}/.timestamp
     COMMAND ${CMAKE_COMMAND} -E remove_directory ${PADDLE_PYTHON_BUILD_DIR}/lib-python
     COMMAND ${CMAKE_COMMAND} -E copy_directory ${PADDLE_PYTHON_BUILD_DIR}/lib* ${PADDLE_PYTHON_BUILD_DIR}/lib-python
-    DEPENDS gen_proto_py copy_paddle_pybind framework_py_proto profiler_py_proto ${PY_FILES} ${external_project_dependencies} ${COPY_PADDLE_MASTER})
+    # FIXME(gongwb) It seems confusing or is a bug of cmake: can't get dependency of paddle_pybind.
+    DEPENDS paddle_pybind gen_proto_py copy_paddle_pybind framework_py_proto profiler_py_proto ${PY_FILES} ${external_project_dependencies} ${COPY_PADDLE_MASTER})
```

Comment:
@luotao1 Any idea why paddle_pybind is needed? It seems copy_paddle_pybind depends on core.so, and core.so is essentially copied from paddle_pybind?

Reply:
In theory, yes。
But in the generated makefile, I can't find this dependency.  I try many tests, the denpendt chains cannot be passed.So I add it manually.

Reply:
I guess the reason is that #9512 move the `core.so` from `PADDLE_SOURCE_DIR` to `PADDLE_BINARY_DIR`
![image](https://user-images.githubusercontent.com/6836917/38966443-e4751e14-43b4-11e8-9da9-7e290fd74afa.png)
@wanglei828 Could you help see this problem?

Reply:
I will try again.